06.bowling_object/lib/frame.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require_relative 'shot'
+
+class Frame
+  def initialize(idx, first_mark, second_mark = nil, third_mark = nil)
+    @idx = idx
+    @shots = [Shot.new(first_mark), Shot.new(second_mark), Shot.new(third_mark)]
+  end
+
+  def score(frames)
+    final? ? score_without_bonus : score_without_bonus + score_bonus(frames)
```

Comment:
```suggestion
    score_without_bonus + score_bonus(frames)
```

finalかどうかはscore_bonusに判定させればいいのでは？と思いました